src/test/java/jenkins/jmh/benchmarks/CascStateBenchmarks.java

```diff
@@ -19,6 +20,8 @@ protected String getResourcePath() {
 
     @Benchmark
     public void benchmark(MyState state) {
-        Objects.requireNonNull(state.getJenkins());
+        assertEquals(state.getJenkins().getSystemMessage(),
+                "Benchmark started with Configuration as Code");
+        assertEquals(state.getJenkins().getNumExecutors(), 22);
```

Comment:
To avoid any extra dependency, the benchmark just checks if the system message is correctly set or not.

Reply:
It is fine as a test benchmark